src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
-public class MainApp {
+public static void main(String[] args) {
+Machine[] machines = {
+        new Truck(),
+        new Bulldozer(),
+        new Excavator()
+};
 
+for (Machine machine : machines){
+    machine.doWork();
+    machine.stopWork();
+}
```

Comment:
The `doWork` and `stopWork` methods should print informative messages indicating what type of machine is working or stopping. Currently, it's not clear if these methods are implemented to do so. Ensure that the `doWork` and `stopWork` methods in `Truck`, `Bulldozer`, and `Excavator` classes print out messages that specify the machine type.